src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -27,4 +27,20 @@ public Integer getLength() {
     public String getColor() {
         return color;
     }
+
+    @Override
+    public boolean equals(Object obj) {
+        return obj instanceof Rectangle && (this.getColor().equals(((Rectangle) obj).getColor())
```

Comment:
should we use it instanceof or getClass() and why? 
+ there should be 4 checks in total, you have 2 now